src/shared/components/Buttons.js

```diff
@@ -0,0 +1,133 @@
+import classnames from 'classnames';
```

Comment:
We've discussed this on a surface level as a team, but here's the actual proposed shared Button components...